src/net.cpp

```diff
@@ -2397,6 +2370,9 @@ bool CConnman::Start(CScheduler& scheduler, const Options& connOptions)
         LOCK(m_cs_interrupt_event);
         m_interrupt_event = event_new(m_event_base, -1, 0, [](evutil_socket_t, short, void* data) {static_cast<CConnman*>(data)->InterruptEvents();}, (this));
     }
+    m_event_timeout = timeval{60, 0};
+    const timeval* common_timeout = event_base_init_common_timeout(m_event_base, &m_event_timeout);
+    memcpy(&m_event_timeout, common_timeout, sizeof(m_event_timeout));
```

Comment:
The documentation on `event_base_init_common_timeout` says "To do this, call this function with the common duration. It will return a pointer to a different, opaque timeout value. (Don't depend on its actual contents!)".

I'm not sure whether that means you need to use the exact pointer returned, or just a copy of its value (as this code does).

Reply:
Good eye! I wasn't sure how to store this properly either. This usage is taken from the official documentation. I'll add a comment about its sanity and a link to the docs.